src/pages/stablecoins.js

```diff
@@ -0,0 +1,882 @@
+    ],
+    cons: [
+      "Centralized – someone must issue the tokens.",
+      "Requires auditing to ensure company has suffficient reserves.",
+    ],
+    projects: ["Tether", "USDc"],
+    links: ["https://www.tether.com", "https://www.tether.com"],
+  },
+  {
+    title: "Crypto backed",
+    description:
+      "These stablecoins are backed by other crypto assets, like ETH. Their price depends on the value of the underlying asset (or collateral), which can be volatile. Because ETH's value can fluctuate, these stablecoins are overcollateralised to ensure the price stays as stable as possible. This means it's closer to say that a $1 crypto backed stablecoin has an underlying crypto asset worth at least $2. So if the price of ETH drops, more ETH must be used to back the stablecoin, else the stablecoins will lose their value.",
+    emoji: ":unicorn:",
+    pros: [
+      "Transparent and fully decentralized.",
+      "Quick to turn into other crypto assets.",
+    ],
+    cons: [
+      "Less stable than fiat-backed stablecoins.",
+      "You need to keep an eye on the value of the crypto collateral.",
+    ],
+    projects: ["Dai", "Test"],
+    links: ["https://www.tether.com", "https://www.tether.com"],
+  },
+  {
+    title: "Precious metals ",
+    description:
+      "Like fiat-backed coins, instead these stablecoins use resources like Gold to maintain their value.",
+    emoji: ":gem_stone:",
+    pros: [
+      "Safe against crypto volatility.",
+      "Safe if blockchain is attacked.",
+    ],
+    cons: [
+      "Centralized – someone must issue the tokens.",
+      "You need to trust the token issuer and the precious metal reserves.",
+      "Expensive and slow to turn back into fiat currency.",
+    ],
+    projects: ["Dai", "Test"],
+    links: ["https://www.tether.com", "https://www.tether.com"],
+  },
+  {
+    title: "Non-collateralised",
+    description:
+      "These stablecoins aren't backed by any other asset. Instead an algorithm will sell tokens if the price falls below the desired value and supply tokens if the value goes beyond the desired amount. Because the number of these tokens in circulation changes regularly, the number of tokens you own will change, but will always reflect your share.",
+    emoji: ":chart_with_downwards_trend:",
+    pros: ["No collateral needed.", "Controlled by a public algorithm."],
+    cons: [
+      "You need to trust (or be able to read) the algorithm.",
+      "Your balance of coins will fluctuate.",
+    ],
+    projects: ["Dai", "Test"],
+    links: ["https://www.tether.com", "https://www.tether.com"],
+  },
+]
+
+const tokens = [
+  {
+    emoji: ":globe_showing_americas:",
+    description:
+      "Accept them wherever you live – no bank account or personal details required.",
+  },
+  {
+    emoji: ":shield:",
+    description:
+      "Stablecoins are censor-proof and anonymous. No one can stop you receiving them, sending them, or swapping them.",
```

Comment:
But I think some CAN censor, e.g. https://cointelegraph.com/news/centre-freezes-ethereum-address-holding-100k-usdc

Reply:
Oh interesting. Maybe we just remove this point then.